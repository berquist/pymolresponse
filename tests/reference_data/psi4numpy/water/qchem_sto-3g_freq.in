$rem
  jobtype = freq
  method = hf
  basis = sto-3g
  scf_convergence = 9
  thresh = 14
  symmetry = false
  sym_ignore = true
$end

$molecule
0 1
O          0.000000000000     0.000000000000    -0.075791843589
H          0.000000000000    -0.866811828967     0.601435779270
H          0.000000000000     0.866811828967     0.601435779270
$end
