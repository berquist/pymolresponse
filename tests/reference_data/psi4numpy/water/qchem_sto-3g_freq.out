gold 127.0.0
Q-Chem Developer Version! Running on 127.0.0.*
                  Welcome to Q-Chem
     A Quantum Leap Into The Future Of Chemistry


 Q-Chem 5.1, Q-Chem, Inc., Pleasanton, CA (2018)

 Yihan Shao,  Zhengting Gan,  E. Epifanovsky,  A. T. B. Gilbert,  M. Wormit,  
 J. Kussmann,  A. W. Lange,  A. Behn,  Jia Deng,  Xintian Feng,  D. Ghosh,  
 M. Goldey,  P. R. Horn,  L. D. Jacobson,  I. Kaliman,  T. Kus,  A. Landau,  
 Jie Liu,  E. I. Proynov,  R. M. Richard,  R. P. Steele,  E. J. Sundstrom,  
 H. L. Woodcock III,  P. M. Zimmerman,  D. Zuev,  B. Albrecht,  E. Alguire,  
 S. A. Baeppler,  D. Barton,  Z. Benda,  Y. A. Bernard,  E. J. Berquist,  
 K. B. Bravaya,  H. Burton,  D. Casanova,  Chun-Min Chang,  Yunqing Chen,  
 A. Chien,  K. D. Closser,  M. P. Coons,  S. Coriani,  S. Dasgupta,  
 A. L. Dempwolff,  M. Diedenhofen,  Hainam Do,  R. G. Edgar,  Po-Tung Fang,  
 S. Faraji,  S. Fatehi,  Qingguo Feng,  K. D. Fenk,  J. Fosso-Tande,  
 Qinghui Ge,  A. Ghysels,  G. Gidofalvi,  J. Gomes,  J. Gonthier,  A. Gunina,  
 D. Hait,  M. W. D. Hanson-Heine,  P. H. P. Harbach,  A. W. Hauser,  
 J. E. Herr,  E. G. Hohenstein,  Z. C. Holden,  Kerwin Hui,  B. C. Huynh,  
 T.-C. Jagau,  Hyunjun Ji,  B. Kaduk,  K. Khistyaev,  Jaehoon Kim,  
 P. Klunzinger,  K. Koh,  D. Kosenkov,  L. Koulias,  T. Kowalczyk,  
 C. M. Krauter,  A. Kunitsa,  Ka Un Lao,  A. Laurent,  K. V. Lawler,  
 Joonho Lee,  D. Lefrancois,  S. Lehtola,  D. S. Levine,  Yi-Pei Li,  
 You-Sheng Lin,  Fenglai Liu,  E. Livshits,  A. Luenser,  P. Manohar,  
 E. Mansoor,  S. F. Manzer,  Shan-Ping Mao,  Yuezhi Mao,  N. Mardirossian,  
 A. V. Marenich,  T. Markovich,  L. A. Martinez-Martinez,  S. A. Maurer,  
 N. J. Mayhall,  S. C. McKenzie,  J.-M. Mewes,  A. F. Morrison,  
 J. W. Mullinax,  K. Nanda,  T. S. Nguyen-Beck,  R. Olivares-Amaya,  
 J. A. Parkhill,  T. M. Perrine,  F. Plasser,  P. Pokhilko,  S. Prager,  
 A. Prociuk,  E. Ramos,  D. R. Rehn,  F. Rob,  M. Schneider,  N. Sergueev,  
 S. M. Sharada,  S. Sharma,  D. W. Small,  T. Stauch,  T. Stein,  
 Yu-Chuan Su,  A. J. W. Thom,  A. Tkatchenko,  T. Tsuchimochi,  N. M. Tubman,  
 L. Vogt,  M. L. Vidal,  O. Vydrov,  M. A. Watson,  J. Wenzel,  
 M. de Wergifosse,  T. A. Wesolowski,  A. White,  J. Witte,  A. Yamada,  
 Jun Yang,  K. Yao,  S. Yeganeh,  S. R. Yost,  Zhi-Qiang You,  A. Zech,  
 Igor Ying Zhang,  Xing Zhang,  Yan Zhao,  Ying Zhu,  B. R. Brooks,  
 G. K. L. Chan,  C. J. Cramer,  M. S. Gordon,  W. J. Hehre,  A. Klamt,  
 M. W. Schmidt,  C. D. Sherrill,  D. G. Truhlar,  A. Aspuru-Guzik,  R. Baer,  
 A. T. Bell,  N. A. Besley,  Jeng-Da Chai,  R. A. DiStasio Jr.,  A. Dreuw,  
 B. D. Dunietz,  T. R. Furlani,  Chao-Ping Hsu,  Yousung Jung,  Jing Kong,  
 D. S. Lambrecht,  WanZhen Liang,  C. Ochsenfeld,  V. A. Rassolov,  
 L. V. Slipchenko,  J. E. Subotnik,  T. Van Voorhis,  J. M. Herbert,  
 A. I. Krylov,  P. M. W. Gill,  M. Head-Gordon

 Contributors to earlier versions of Q-Chem not listed above: 
 R. D. Adamson,  B. Austin,  J. Baker,  G. J. O. Beran,  K. Brandhorst,  
 S. T. Brown,  E. F. C. Byrd,  A. K. Chakraborty,  C.-L. Cheng,  
 Siu Hung Chien,  D. M. Chipman,  D. L. Crittenden,  H. Dachsel,  
 R. J. Doerksen,  A. D. Dutoi,  L. Fusti-Molnar,  W. A. Goddard III,  
 A. Golubeva-Zadorozhnaya,  S. R. Gwaltney,  G. Hawkins,  A. Heyden,  
 S. Hirata,  G. Kedziora,  F. J. Keil,  C. Kelley,  Jihan Kim,  R. A. King,  
 R. Z. Khaliullin,  P. P. Korambath,  W. Kurlancheek,  A. M. Lee,  M. S. Lee,  
 S. V. Levchenko,  Ching Yeh Lin,  D. Liotard,  R. C. Lochan,  I. Lotan,  
 P. E. Maslen,  N. Nair,  D. P. O'Neill,  D. Neuhauser,  E. Neuscamman,  
 C. M. Oana,  R. Olson,  B. Peters,  R. Peverati,  P. A. Pieniazek,  
 Y. M. Rhee,  J. Ritchie,  M. A. Rohrdanz,  E. Rosta,  N. J. Russ,  
 H. F. Schaefer III,  N. E. Schultz,  N. Shenvi,  A. C. Simmonett,  A. Sodt,  
 D. Stuck,  K. S. Thanthiriwatte,  V. Vanovschi,  Tao Wang,  A. Warshel,  
 C. F. Williams,  Q. Wu,  X. Xu,  W. Zhang

 Please cite Q-Chem as follows:
 Y. Shao et al., Mol. Phys. 113, 184-215 (2015)
 DOI: 10.1080/00268976.2014.952696

 Q-Chem 5.1.0 for Intel X86 EM64T Linux

 Parts of Q-Chem use Armadillo 8.300.2 (Tropical Shenanigans).
 http://arma.sourceforge.net/

Compilation information
-----------------------

 SVN revision             | 28616
 SVN branch               | branches/libresponse
 Last SVN revision        | 28616
 Last SVN author          | ericb
 Hostname                 | gold
 User                     | eric
 System                   | Linux-4.17.2-1-ARCH
 Processor                | x86_64
 Compiler                 | GNU
 C compiler version       | 8.1.1
 CXX compiler version     | 8.1.1
 Fortran compiler version | 8.1.1
 Configuration time       | 2018-07-07 14:55:32


 Q-Chem begins on Sun Jul 15 10:35:11 2018  

Host: gold
0

     Scratch files written to /tmp/qchem/qchem20959//
Input Ideriv = 2
Curr. Ideriv = -1
Max.  Ideriv = 2

Checking the input file for inconsistencies... 	...done.

--------------------------------------------------------------
User input:
--------------------------------------------------------------
$rem
  jobtype = freq
  method = hf
  basis = sto-3g
  scf_convergence = 9
  thresh = 14
  symmetry = false
  sym_ignore = true
$end

$molecule
0 1
O          0.000000000000     0.000000000000    -0.075791843589
H          0.000000000000    -0.866811828967     0.601435779270
H          0.000000000000     0.866811828967     0.601435779270
$end
--------------------------------------------------------------
 ----------------------------------------------------------------
             Standard Nuclear Orientation (Angstroms)
    I     Atom           X                Y                Z
 ----------------------------------------------------------------
    1      O       0.0000000000     0.0000000000    -0.0757918436
    2      H       0.0000000000    -0.8668118290     0.6014357793
    3      H       0.0000000000     0.8668118290     0.6014357793
 ----------------------------------------------------------------
 Nuclear Repulsion Energy =     8.0023664841 hartrees
 There are        5 alpha and        5 beta electrons
 Requested basis set is STO-3G
 There are 4 shells and 7 basis functions

 Total QAlloc Memory Limit   2000 MB
 Mega-Array Size       188 MB
 MEM_STATIC part       192 MB

                       Distance Matrix (Angstroms)
             O (  1)   H (  2)
   H (  2)  1.100000
   H (  3)  1.100000  1.733624

 A cutoff of  1.0D-14 yielded     10 shell pairs
 There are        34 function pairs
 Smallest overlap matrix eigenvalue = 4.34E-01
 Guess from superposition of atomic densities
 Warning:  Energy on first SCF cycle will be non-variational
 SAD guess density has 10.000000 electrons

 -----------------------------------------------------------------------
  General SCF calculation program by
  Eric Jon Sundstrom, Paul Horn, Yuezhi Mao, Dmitri Zuev, Alec White,
  David Stuck, Shaama M.S., Shane Yost, Joonho Lee, David Small,
  Daniel Levine, Susi Lehtola, Hugh Burton, Evgeny Epifanovsky
 -----------------------------------------------------------------------
 Hartree-Fock
 A restricted SCF calculation will be
 performed using DIIS
 SCF converges when DIIS error is below 1.0e-09
 ---------------------------------------
  Cycle       Energy         DIIS error
 ---------------------------------------
    1     -74.3632364404      3.11e-01  
    2     -74.8897018458      7.23e-02  
    3     -74.9408834963      1.22e-02  
    4     -74.9420558345      1.59e-03  
    5     -74.9420798796      3.35e-05  
    6     -74.9420798949      1.29e-05  
    7     -74.9420798988      2.55e-08  
    8     -74.9420798988      5.80e-10  Convergence criterion met
 ---------------------------------------
 SCF time:   CPU 0.34s  wall 0.00s 
 SCF   energy in the final basis set =      -74.9420798988
 Total energy in the final basis set =      -74.9420798988

 --------------------------------------------------------------

                    Orbital Energies (a.u.)
 --------------------------------------------------------------

 Alpha MOs
 -- Occupied --
-20.263  -1.210  -0.548  -0.437  -0.388
 -- Virtual --
  0.478   0.588
 --------------------------------------------------------------

          Ground-State Mulliken Net Atomic Charges

     Atom                 Charge (a.u.)
  ----------------------------------------
      1 O                    -0.253146
      2 H                     0.126573
      3 H                     0.126573
  ----------------------------------------
  Sum of atomic charges =     0.000000

 -----------------------------------------------------------------
                    Cartesian Multipole Moments
 -----------------------------------------------------------------
    Charge (ESU x 10^10)
                 0.0000
    Dipole Moment (Debye)
         X       0.0000      Y       0.0000      Z       1.5340
       Tot       1.5340
    Quadrupole Moments (Debye-Ang)
        XX      -6.2076     XY      -0.0000     YY      -4.3168
        XZ      -0.0000     YZ       0.0000     ZZ      -5.2054
    Octopole Moments (Debye-Ang^2)
       XXX       0.0000    XXY       0.0000    XYY       0.0000
       YYY       0.0000    XXZ      -0.4476    XYZ      -0.0000
       YYZ       0.3916    XZZ       0.0000    YZZ       0.0000
       ZZZ      -0.9572
    Hexadecapole Moments (Debye-Ang^3)
      XXXX      -3.3591   XXXY      -0.0000   XXYY      -2.1047
      XYYY      -0.0000   YYYY      -7.9263   XXXZ      -0.0000
      XXYZ       0.0000   XYYZ      -0.0000   YYYZ       0.0000
      XXZZ      -1.5957   XYZZ      -0.0000   YYZZ      -1.9038
      XZZZ      -0.0000   YZZZ       0.0000   ZZZZ      -5.8351
 -----------------------------------------------------------------
 Calculating MO derivatives via CPSCF
   1        10           2      0.007415    0.005093
   2        12           0      0.000000    0.000000    Roots Converged
 Time for AOints: 0.0 s (CPU) 0.0 s (wall)
 ------------------------------------------------------------------------------
                     AOints        : Timing summary (seconds)
 ------------------------------------------------------------------------------
     job step          cpu (% of tot)      sys (% of tot)     wall (% of tot)
 ------------------------------------------------------------------------------
  cpscf_lag         0.4301E-02 (66.3)   0.0000E+00 ( 0.0)   0.9467E-02 (70.9)
  iterzp            0.1332E-02 (20.5)   0.0000E+00 ( 0.0)   0.1604E-02 (12.0)
  make_fx           0.0000E+00 ( 0.0)   0.0000E+00 ( 0.0)   0.0000E+00 ( 0.0)
  AOints            0.4960E-03 ( 7.6)   0.1000E-05 (****)   0.4953E-03 ( 3.7)
 ------------------------------------------------------------------------------
   Grand Totals     0.6490E-02          0.1000E-05          0.1336E-01
 ------------------------------------------------------------------------------
 Calculating analytic Hessian of the SCF energy
 Polarizability Matrix (a.u.)
            1           2           3
    1  -0.0503862   0.0000000  -0.0000000
    2   0.0000000  -7.9355640   0.0000000
    3  -0.0000000   0.0000000  -3.0682113

 Direct stationary perturbation theory relativistic correction:

 rels  =       0.031240218577
 relv  =      -0.096618759725
 rel2e =       0.023053417282
 E_rel =      -0.042325123867

 Hessian of the SCF Energy
            1           2           3           4           5           6
    1   0.0761395   0.0000000   0.0000000  -0.0380698   0.0000000   0.0000000
    2   0.0000000   0.4829054  -0.0000000  -0.0000000  -0.2414527   0.1589002
    3   0.0000000  -0.0000000   0.4373450  -0.0000000   0.0734423  -0.2186725
    4  -0.0380698  -0.0000000  -0.0000000   0.0453774   0.0000000   0.0000000
    5   0.0000000  -0.2414527   0.0734423   0.0000000   0.2578650  -0.1161712
    6   0.0000000   0.1589002  -0.2186725   0.0000000  -0.1161712   0.1977520
    7  -0.0380698  -0.0000000  -0.0000000  -0.0073077  -0.0000000  -0.0000000
    8  -0.0000000  -0.2414527  -0.0734423   0.0000000  -0.0164123  -0.0427289
    9  -0.0000000  -0.1589002  -0.2186725  -0.0000000   0.0427289   0.0209205
            7           8           9
    1  -0.0380698  -0.0000000  -0.0000000
    2  -0.0000000  -0.2414527  -0.1589002
    3  -0.0000000  -0.0734423  -0.2186725
    4  -0.0073077   0.0000000  -0.0000000
    5  -0.0000000  -0.0164123   0.0427289
    6  -0.0000000  -0.0427289   0.0209205
    7   0.0453774   0.0000000   0.0000000
    8   0.0000000   0.2578650   0.1161712
    9   0.0000000   0.1161712   0.1977520
 Gradient time:  CPU 0.02 s  wall 0.03 s
 **********************************************************************
 **                                                                  **
 **                       VIBRATIONAL ANALYSIS                       **
 **                       --------------------                       **
 **                                                                  **
 **        VIBRATIONAL FREQUENCIES (CM**-1) AND NORMAL MODES         **
 **     FORCE CONSTANTS (mDYN/ANGSTROM) AND REDUCED MASSES (AMU)     **
 **                  INFRARED INTENSITIES (KM/MOL)                   **
 **                                                                  **
 **********************************************************************


 Mode:                 1                      2                      3
 Frequency:      2178.88                2983.27                3236.37
 Force Cnst:      3.0045                 5.5230                 6.6733
 Red. Mass:       1.0741                 1.0533                 1.0814
 IR Active:          YES                    YES                    YES
 IR Intens:        5.109                 40.510                 37.266
 Raman Active:       YES                    YES                    YES
               X      Y      Z        X      Y      Z        X      Y      Z
 O          0.000 -0.000 -0.067   -0.000 -0.000 -0.055   -0.000 -0.070  0.000
 H         -0.000  0.468  0.528   -0.000 -0.555  0.437   -0.000  0.556 -0.434
 H         -0.000 -0.468  0.528   -0.000  0.555  0.437   -0.000  0.556  0.434
 TransDip  -0.000  0.000  0.072   -0.000  0.000 -0.204    0.000 -0.196  0.000

 STANDARD THERMODYNAMIC QUANTITIES AT   298.15 K  AND     1.00 ATM

   This Molecule has  0 Imaginary Frequencies
   Zero point vibrational energy:       12.006 kcal/mol

   Atom    1 Element O  Has Mass   15.99491
   Atom    2 Element H  Has Mass    1.00783
   Atom    3 Element H  Has Mass    1.00783
   Molecular Mass:    18.010570 amu
   Principal axes and moments of inertia in amu*Bohr^2:
                             1           2           3
    Eigenvalues --        2.93183     5.40835     8.34018
          X               0.00000     0.00000     1.00000
          Y               1.00000     0.00000     0.00000
          Z               0.00000     1.00000     0.00000
   Rotational Symmetry Number is   1
   The Molecule is an Asymmetric Top
   Translational Enthalpy:        0.889 kcal/mol
   Rotational Enthalpy:           0.889 kcal/mol
   Vibrational Enthalpy:         12.006 kcal/mol
   gas constant (RT):             0.592 kcal/mol
   Translational Entropy:        34.608  cal/mol.K
   Rotational Entropy:           12.656  cal/mol.K
   Vibrational Entropy:           0.001  cal/mol.K

   Total Enthalpy:               14.376 kcal/mol
   Total Entropy:                47.265  cal/mol.K

 ======================================================================

 STANDARD THERMODYNAMIC QUANTITIES FOR ANHARMONIC VIBRATION -- TOSH

   Zero point vibrational energy: ************ kcal/mol

   Vibrational Enthalpy:       Infinity kcal/mol
   Vibrational Entropy:        Infinity  cal/mol.K

   Total Enthalpy:             Infinity kcal/mol
   Total Entropy:              Infinity  cal/mol.K

 ======================================================================

 STANDARD THERMODYNAMIC QUANTITIES FOR ANHARMONIC VIBRATION -- VPT2

   Zero point vibrational energy: ************ kcal/mol

   Vibrational Enthalpy:       Infinity kcal/mol
   Vibrational Entropy:        Infinity  cal/mol.K

   Total Enthalpy:             Infinity kcal/mol
   Total Entropy:              Infinity  cal/mol.K

 ======================================================================

Archival summary:
1\1\gold\FREQ\HF\STO-3G\12\eric\SunJul1510:35:112018SunJul1510:35:112018\0\\#,FREQ,HF,STO-3G,\\0,1\O\H,1,1.1\H,1,1.1,2,104\\HF=-74.9420799\\@

 Total job time:  0.45s(wall), 0.42s(cpu) 
 Sun Jul 15 10:35:11 2018

        *************************************************************
        *                                                           *
        *  Thank you very much for using Q-Chem.  Have a nice day.  *
        *                                                           *
        *************************************************************


